package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "fsevents": "^2.1.2",
```

Comment:
this dependancy was causing us issues with netlify. can you please remove it from package.json?